### ISSUE TITLE
Markdown parser doesn't work with quarto files 

### ISSUE THREAD

**Issue by maikol-solis** [emacs-tree-sitter/tree-sitter-langs#564]
Hi! 

I'm trying to visualize some quarto files, which are basically markdown with some R or python chunks to execute data analysis. 

The problem is if I display the same text in a `.md` and a `.qmd` the results are entirely different. 

`.md`
![image](https://github.com/emacs-tree-sitter/tree-sitter-langs/assets/4078489/5fa61bab-bd88-4dbe-90af-877decfd9b1e)

`.qmd`
![image](https://github.com/emacs-tree-sitter/tree-sitter-langs/assets/4078489/1b314405-739d-4689-92aa-134ae37133ed)

Why is there a difference, and how can we fix it?

Thanks.

**Comment by jcs090218** [comment]
Is the markdown mode being used in the qmd file? 🤔

**Comment by maikol-solis** [comment]
Yes, it uses `poly-quarto-mode` (https://github.com/quarto-dev/quarto-emacs). 

I have recorded exactly what I am seeing on my laptop. 

https://github.com/emacs-tree-sitter/tree-sitter-langs/assets/4078489/e5557744-8a2d-43db-9982-2a91e5b3bb17

**Comment by jcs090218** [comment]
Hmmm... would adding `poly-quarto-mode` here work? 🤔 

https://github.com/emacs-tree-sitter/tree-sitter-langs/blob/ccb5529406fe2a1f9e54d6425ebf4931cd4c59c7/tree-sitter-langs.el#L154

**Comment by maikol-solis** [comment]
I have this in my config.el (doom-emacs) 

```elisp 
(after! tree-sitter
  (add-to-list 'tree-sitter-major-mode-language-alist '(poly-quarto-mode . markdown)))
```

![image](https://github.com/emacs-tree-sitter/tree-sitter-langs/assets/4078489/6557b596-5713-47c4-93b4-e7ca5bd56faa)

I don't know if I need to change something else...???
